src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let cost = 0;
+
+  for (let i = 0; i <= days; i++) {
+    if (days < 3) {
+      cost = 40 * days;
+    } else if (days >= 3 && days < 7) {
+      cost = 40 * days - 20;
+    } else {
+      cost = 40 * days - 50;
```

Comment:
20 and 50 look like 'magic' numbers. Move them to variables and give them proper names.